tasks/extract_filepaths_for_granule/extract_filepaths_for_granule.py

```diff
@@ -121,14 +120,15 @@ def get_regex_buckets(event):
         # {'regex': '.*.cmr.json$', 'sampleFileName': 'L0A_0420.cmr.json', 'bucket': 'public'}]
         regex_buckets = {}
         for regx in file_buckets:
-            regex_buckets[regx["regex"]] = buckets[regx["bucket"]]
+            regex_buckets[regx["regex"]] = buckets[regx["bucket"]["name"]]
 
         # regex_buckets example:
         # {'.*.h5$': 'podaac-sndbx-cumulus-protected',
         #  '.*.iso.xml$': 'podaac-sndbx-cumulus-protected',
         #  '.*.h5.mp$': 'podaac-sndbx-cumulus-public',
         #  '.*.cmr.json$': 'podaac-sndbx-cumulus-public'}
     except KeyError as err:
+        level = "event['config']"
```

Comment:
I would add a log error here before the raise so it is captured in the logs.

Reply:
added `LOGGER.error(f'KeyError: "{level}[{str(err)}]" is required')`